src/dashboard/src/components/backlog/views.py

```diff
@@ -0,0 +1,216 @@
+
+    :param request: The Django request object
+    :return: The main backlog page rendered
+    """
+    check_and_remove_deleted_transfers()
+    return render(request, 'backlog/backlog.html', locals())
+
+
+def get_es_property_from_column_index(index, file_mode):
+    """
+    When the user clicks a column header in the data table, we'll receive info in the ajax request
+    telling us which column # we're supposed to sort across in our query. This function will translate
+    the column index to the corresponding property name we'll tell ES to sort on.
+
+    :param index: The column index that the data table says we're sorting on
+    :param file_mode: Whether we're looking at transfers or transfer files
+    :return: The ES document property name corresponding to the column index in the data table.
+    """
+    table_columns = (
+        ('name', 'uuid', 'file_count', 'ingest_date', None),     # Transfers are being displayed
+        ('filename', 'sipuuid', None)                            # Transfer files are being displayed
+    )
+
+    if index < 0 or index >= len(table_columns[file_mode]):
+        raise IndexError('Column index specified is invalid, got {}'.format(index))
+
+    return table_columns[file_mode][index]
+
+
+@decorators.elasticsearch_required()
+def search(request):
+    """
+    A JSON end point that returns results for various backlog transfers and their files.
+
+    :param request: The Django request object
+    :return: A JSON object including required metadata for the datatable and the backlog search results.
+    """
+    # get search parameters from request
+    queries, ops, fields, types = advanced_search.search_parameter_prep(request)
+
+    file_mode = True if request.GET.get('file_mode') == 'true' else False
+    page_size = int(request.GET.get('iDisplayLength', 10))
+    start = int(request.GET.get('iDisplayStart', 0))
+
+    order_by = get_es_property_from_column_index(int(request.GET.get('iSortCol_0', 0)), file_mode)
+    sort_direction = request.GET.get('sSortDir_0', 'asc')
+
+    conn = elasticSearchFunctions.connect_and_create_index('transfers')
+
+    if 'query' not in request.GET:
+        queries, ops, fields, types = (['*'], ['or'], [''], ['term'])
+
+    query = advanced_search.assemble_query(queries, ops, fields, types, search_index='transfers',
+                                           doc_type='transferfile', filters={'term': {'status': 'backlog'}})
+    try:
+        doc_type = 'transferfile' if file_mode else 'transfer'
+
+        hit_count = conn.search(index='transfers', doc_type=doc_type, body=query, search_type='count')['hits']['total']
+        hits = conn.search(
+            index='transfers',
+            doc_type=doc_type,
+            body=query,
+            from_=start,
+            size=page_size,
+            sort=order_by + ':' + sort_direction if order_by else ''
+        )
+
+    except Exception as e:
+        logger.exception('Error accessing index: {}'.format(e))
```

Comment:
`logger.exception` prints the full stacktrace for the exception, so printing it in the message is unnecessary.  The descriptive message of what was happening when the exception happened is useful.